tools/integration_tests/util/setup/setup.go

```diff
@@ -327,3 +327,13 @@ func CleanupDirectoryOnGCS(directoryPathOnGCS string) {
 			directoryPathOnGCS, err)
 	}
 }
+
+func RunMountedDirectoryTestsWithTestBucket() bool {
```

Comment:
can you rename this method AreBothMountedDirectoryAndTestBucketFlags are set. Lets keep the methods generic